config/deploy.rb

```diff
@@ -157,6 +157,36 @@
 
   after :finishing, 'deploy:cleanup'
 
+  #   # You can/ should apply this command to a subset of hosts
+# cap --hosts=lib-approvals-staging2.lib.princeton.edu staging application:remove_from_nginx
+desc "Marks the server(s) to be removed from the loadbalancer"
```

Comment:
I see what happened - you accidentally left off the first line - which will then change the formatting for everything else, so rubocop will yell. 

```suggestion
namespace :application do
  desc "Marks the server(s) to be removed from the loadbalancer"
```